### ISSUE TITLE
pgnet印章训练了很久没有效果

### ISSUE THREAD

**Issue by shiyun123456789** [PaddlePaddle/PaddleOCR#9895]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：linux gpu 11.2 
- 版本号/Version：Paddle： 2.4.2.post112  PaddleOCR：2.6.1.00 问题相关组件/Related components：
- 运行指令/Command Code： python3.7 tools/train.py -c configs/e2e/e2e_r50_vd_pg.yml
- 完整报错/Complete Error Message：
- gpu16G显存，所以bach_size设置为了4，训练文件采用的都是14点标注。
- 开始使用30张图片，可以得到印章弯曲文字的框，训练集调大到500张图片后，训练了很多天都不能框到文字。
- 在gpu环境下准备了五百张印章弯曲文字识别，训练了很多天没有效果。
- 印章图片：
- 
![1683546166016](https://user-images.githubusercontent.com/31406086/236815300-282bb82f-6dd8-4cd9-8eda-06586a608b8e.jpg)

![0032799802917](https://user-images.githubusercontent.com/31406086/236814098-cc1b421d-841a-4447-981a-95624f68a59c.jpg)
效果：
![9981651698977](https://user-images.githubusercontent.com/31406086/236816170-f3de0761-bca9-4f4b-8320-d019e12005ed.jpg)

训练配置：
Global:
  use_gpu: True
  epoch_num: 10000
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/pgnet_r50_vd_totaltext/test
  save_epoch_step: 10
  # evaluation is run every 0 iterationss after the 1000th iteration
  eval_batch_step: [ 0, 1000 ]
  cal_metric_during_train: False
  pretrained_model: 
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img:
  infer_visual_type: CN # two mode: EN is for english datasets, CN is for chinese datasets
  valid_set: totaltext # two mode: totaltext valid curved words, partvgg valid non-curved words
  save_res_path: ./output/pgnet_r50_vd_totaltext/test/predicts_pgnet.txt
  character_dict_path: ppocr/utils/ppocr_keys_v1.txt
  character_type: CN
  max_text_length: 30 # the max length in seq
  max_text_nums: 10 # the max seq nums in a pic
  tcl_len: 64

Architecture:
  model_type: e2e
  algorithm: PGNet
  Transform:
  Backbone:
    name: ResNet
    layers: 50
  Neck:
    name: PGFPN
  Head:
    name: PGHead
    character_dict_path: ppocr/utils/ppocr_keys_v1.txt # the same as Global:character_dict_path

Loss:
  name: PGLoss
  tcl_bs: 64
  max_text_length: 30 # the same as Global: max_text_length
  max_text_nums: 10 # the same as Global：max_text_nums
  pad_num: 6623 # the length of dict for pad

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 50
  regularizer:
    name: 'L2'
    factor: 0.0001

PostProcess:
  name: PGPostProcess
  score_thresh: 0.5
  mode: fast   # fast or slow two ways
  point_gather_mode: align # same as PGProcessTrain: point_gather_mode

Metric:
  name: E2EMetric
  mode: A   # two ways for eval, A: label from txt,  B: label from gt_mat
  gt_mat_dir:  ./train_data/total_text/gt  # the dir of gt_mat
  character_dict_path: ppocr/utils/ic15_dict.txt
  main_indicator: f_score_e2e

Train:
  dataset:
    name: PGDataSet
    data_dir: ./train_data/seal/train
    label_file_list: [./train_data/seal/train/train.txt]
    ratio_list: [1.0]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - E2ELabelEncodeTrain:
      - PGProcessTrain:
          batch_size: 4  # same as loader: batch_size_per_card
          use_resize: True
          use_random_crop: False
          min_crop_size: 24
          min_text_size: 3
          max_text_size: 512
          point_gather_mode: align # two mode: align and none, align mode is better than none mode
      - KeepKeys:
          keep_keys: [ 'images', 'tcl_maps', 'tcl_label_maps', 'border_maps','direction_maps', 'training_masks', 'label_list', 'pos_list', 'pos_mask' ] # dataloader will return list in this order
  loader:
    shuffle: True
    drop_last: True
    batch_size_per_card: 4
    num_workers: 0

Eval:
  dataset:
    name: PGDataSet
    data_dir: ./train_data/seal/eval
    label_file_list: [./train_data/seal/eval/eval.txt]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - E2ELabelEncodeTest:
      - E2EResizeForTest:
          max_side_len: 768
      - NormalizeImage:
          scale: 1./255.
          mean: [ 0.485, 0.456, 0.406 ]
          std: [ 0.229, 0.224, 0.225 ]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: [ 'image', 'shape', 'polys', 'texts', 'ignore_tags', 'img_id']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 0

**Comment by LDOUBLEV** [comment]
<img width="346" alt="image" src="https://user-images.githubusercontent.com/26592129/237009793-dd3fd8d4-4870-4ae7-9aa5-96556a87a1fd.png">
建议印章方向大概矫正之后再训练

**Comment by littlesmallrookie** [comment]
统一方向训练，得到的模型，能检测方向旋转的印章吗？

**Comment by shiyun123456789** [comment]
3:采用pdocr-det后再使用pdocr-rec效果会更好
不太理解pdocr-det pdocr-rec，这个是两阶段训练吗？pgnet训练教程中没有提到pdocr-det pdocr-rec,只提了印章检测，印章串联识别。
pdocr-det，pdocr-rec使用教程可以发下吗？

**Comment by tigerdhl** [comment]
> 3:采用pdocr-det后再使用pdocr-rec效果会更好 不太理解pdocr-det pdocr-rec，这个是两阶段训练吗？pgnet训练教程中没有提到pdocr-det pdocr-rec,只提了印章检测，印章串联识别。 pdocr-det，pdocr-rec使用教程可以发下吗？

问题解决了吗，我也是14点弯曲字符检测，训练后，检测效果很差，训练的数据检测都无法检测出来
